Assets/Scripts/PlayerHealth.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.SceneManagement;
+
+public class PlayerHealth : MonoBehaviour
+{// players helath
+    int healthShip = 3;
+
+
+    private void OnTriggerEnter2D(Collider2D collision)
+    {
+        //Debug.Log("Working");
+        healthShip--;
+        // -1 health for each hit.
+    }
+
+    private void Update()
+    {
+        if (healthShip <= 0)
```

Comment:
healthShip could be checked in the OnTriggerEnter2D after you deduct 1 for each hit.  That way you don't have to check it in the update method